CMakeLists.txt

```diff
@@ -238,7 +240,7 @@ IF( NOT BUILD_DOCUMENTATION_ONLY )
   MARK_AS_ADVANCED(CAPTK_BUILD_OPTION)
   MARK_AS_ADVANCED(CAPTK_DISABLE_CWL)
 
-  # MESSAGE( AUTHOR_WARNING "=== [DEBUG] Building CaPTk version=${PROJECT_VERSION}" )
```

Comment:
Could you keep this line? It is a nice sanity check for nonRelease builds.

Reply:
got it